lib/polyfill/patchedmediakeys_apple.js

```diff
@@ -140,6 +126,26 @@ shaka.polyfill.PatchedMediaKeysApple = class {
     return Promise.resolve();
   }
 
+  /**
+   *
+   * @param {!MediaKeyEvent} sourceEvent
```

Comment:
This parameter appears to be unused.  Could it be removed?  Is there a good reason to keep it?

Reply:
It is used at l.146 :)